crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -80,6 +80,15 @@ def maintenance_updates_installed?
       end
     end
 
+    def clusters_healthy?
+      service_object = CrowbarService.new(Rails.logger)
+      cluster_health = service_object.check_cluster_health
+      unless cluster_health.empty?
+        Rails.logger.warn("clusters report some problems: #{cluster_health}")
```

Comment:
See how the hash could look like. 
https://github.com/crowbar/crowbar-core/blob/stable/3.0/crowbar_framework/app/models/crowbar_service.rb#L319
The original idea probably was that the api somehow passes this to UI which can show it the nice way. I'd say logging such hash is good enough, but ok...